sdcm/sct_events/nodetool.py

```diff
@@ -12,52 +12,38 @@
 # Copyright (c) 2021 ScyllaDB
 
 from sdcm.sct_events import Severity
-from sdcm.sct_events.base import SctEvent
+from sdcm.sct_events.base import ContinuousEvent
 
 
-class NodetoolEvent(SctEvent):
+class NodetoolEvent(ContinuousEvent):
     def __init__(self,
-                 type,
-                 subtype,
-                 severity,
-                 options=None,
+                 nodetool_command,
+                 severity=Severity.NORMAL,
                  node=None,
-                 start=None,
-                 end=None,
-                 duration=None,
-                 error=None,
-                 full_traceback=None):  # pylint: disable=redefined-builtin,too-many-arguments
-        super().__init__(severity=severity)
+                 options=None,
+                 publish_event=True):
+        super().__init__(severity=severity, publish_event=publish_event)
 
         # handle the case if command is like "snapshot -kc keyspace1"
-        nodetool_cmd = type.split()[0]
-        if len(type.split()) > 1:
+        cmd_splitted = nodetool_command.split()
+        if len(cmd_splitted) > 1:
             options_to_list = [options] if options else []
-            options = ' '.join(type.split()[1:] + options_to_list)
+            options = ' '.join(cmd_splitted[1:] + options_to_list)
 
-        self.type = nodetool_cmd
-        self.subtype = subtype
+        self.nodetool_command = cmd_splitted[0]
```

Comment:
This part is not safe, we do not check that first element exists.

Reply:
I added validation. But `nodetool_command` is mandatory, so it can't be empty